.devops/llama-cpp.srpm.spec

```diff
@@ -34,13 +37,14 @@ Models are not included in this package and must be downloaded separately.
 %setup -n llama.cpp-master
 
 %build
-make -j
+cmake -B build -DBUILD_SHARED_LIBS=0
+cmake --build build --config Release -j$(nproc)
```

Comment:
If you only want to bundle `llama-server`, add `-t llama-server` to build only that.

Reply:
Yes a good point. I started to package the libs but it got ugly and I was debating spinning off a separate RPM for libs. Open to suggestions.

Reply:
I don't know how this RPM is intended to be used. If it is meant to be something that distributions can use to bundle a llama.cpp package, it is definitely better to use shared libs and bundle multiple version of the CPU backend as I already pointed before.
Honestly I am not sure why this file is here in the first place. It looks like something that should be maintained by the distro maintainer. Can you explain what is the purpose of this file?